.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: go
 go_import_path: github.com/ethereum/go-ethereum
 sudo: false
+arch:
+  - amd64
+  - arch64
```

Comment:
These two entries look like they mess up the pipeline (a `Test` section appeared): https://travis-ci.org/ethereum/go-ethereum/builds/604822629?utm_medium=notification&utm_source=github_status